libraries/c_sdk/standard/ble/src/iot_ble_gap.c

```diff
@@ -396,6 +396,9 @@ BTStatus_t _setAdvData( IotBleAdvertisementParams_t * pAdvParams )
     pParams.ulAppearance = pAdvParams->appearance;
     pParams.ulMaxInterval = pAdvParams->maxInterval;
     pParams.ulMinInterval = pAdvParams->minInterval;
+
+    pParams.usMinAdvInterval = IOT_BLE_ADVERTISING_INTERVAL;
```

Comment:
Why don't these values come in via the Params struct as above?

Reply:
As of now, the low level parameter is only exposed from HAL layer but not through middle ware. This was to keep it open to later support different advertising modes (to save power) with user option to override them.